src/libraries/System.Formats.Nrbf/src/PACKAGE.md

```diff
@@ -0,0 +1,83 @@
+## About
+
+<!-- A description of the package and where one can find more documentation -->
+
+`System.Formats.Nrbf` exposes only one component: [NrbfDecoder](https://learn.microsoft.com/dotnet/api/system.formats.nrbf.nrbfdecoder): a stateless, forward-only decoder class that can decode .NET Remoting Binary Format (NRBF) binary data from a **stream**.
```

Comment:
We added this line to your guide because it helps to conceptualize what this component is for.
 
```suggestion
`System.Formats.Nrbf` exposes only one component: [NrbfDecoder](https://learn.microsoft.com/dotnet/api/system.formats.nrbf.nrbfdecoder): a stateless, forward-only decoder class that can decode .NET Remoting Binary Format (NRBF) binary data from a **stream**.

You can think of [NrbfDecoder](https://learn.microsoft.com/en-us/dotnet/api/system.formats.nrbf.nrbfdecoder) as being the equivalent of using a JSON/XML reader without the deserializer.
```

What do you think?

Reply:
It's a good point, thanks for bringing new point of view!